packages/firestore/test/integration/util/helpers.ts

```diff
@@ -20,16 +20,27 @@ import { DatabaseId, DatabaseInfo } from '../../../src/core/database_info';
 
 import firebase from './firebase_export';
 
+// tslint:disable-next-line:no-any __karma__ is an untyped global
+declare let __karma__: any;
+
 const PROJECT_CONFIG = require('../../../../../config/project.json');
 
 export const DEFAULT_PROJECT_ID = PROJECT_CONFIG.projectId;
 export const ALT_PROJECT_ID = 'test-db2';
 
-// TODO(b/66917388): Reintroduce way to test against localhost.
-const DEFAULT_SETTINGS = {
-  host: 'firestore.googleapis.com',
-  ssl: true
-};
+const DEFAULT_SETTINGS = getDefaultSettings();
+
+function getDefaultSettings(): firestore.Settings {
+  let karma = typeof __karma__ !== 'undefined' ? __karma__ : undefined;
```

Comment:
Prefer `const` to `let`.

Also, I don't understand why this is even necessary. The RHS is equivalent to `__karma__ !== undefined ? __karma__ : undefined` which seems axiomatic.

I think this can go away and the next line be just

```javascript
if (__karma__ && __karma__.config.firestoreSettings) {
```

Reply:
Erg. I should really get tslint reenabled pronto. (fixed let => const)

Regarding the typeof check, I'm defending against the `__karma__` global not existing, in which case trying to reference it directly will cause a JavaScript error, but doing a `typeof` check on it will succeed... because JavaScript 👏 .

TBH, `__karma__` should always exist right now because that's the only way we run our integration tests. So I could also just stop being so defensive.